pkg/csi/service/wcpguest/controller.go

```diff
@@ -199,13 +237,24 @@ func (c *controller) ReloadConfiguration() {
 func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (
 	*csi.CreateVolumeResponse, error) {
 
+	start := time.Now()
+	isFileVolumeRequest := common.IsFileVolumeRequest(ctx, req.GetVolumeCapabilities())
+	var volumeType string
+	if isFileVolumeRequest {
+		volumeType = common.PrometheusFileVolumeType
+	} else {
```

Comment:
nit: You can drop this else{}

Reply:
I did not quite get this comment. I need this else to initialize `volumeType` to PrometheusBlockVolumeType if the create request is for block volume. Another way to solve this is to initialize the `volumeType` to PrometheusBlockVolumeType and remove the else. I do not have a preference here, so let me know.